[2022-07-27 15:02:22,391] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag.accurate manual__2022-07-27T14:58:04.465454+00:00 [queued]>
[2022-07-27 15:02:22,418] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dag.accurate manual__2022-07-27T14:58:04.465454+00:00 [queued]>
[2022-07-27 15:02:22,418] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 15:02:22,418] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-27 15:02:22,418] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 15:02:22,448] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): accurate> on 2022-07-27 14:58:04.465454+00:00
[2022-07-27 15:02:22,458] {standard_task_runner.py:52} INFO - Started process 1393 to run task
[2022-07-27 15:02:22,464] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag', 'accurate', 'manual__2022-07-27T14:58:04.465454+00:00', '--job-id', '41', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp56jscygc', '--error-file', '/tmp/tmpjncclvqi']
[2022-07-27 15:02:22,465] {standard_task_runner.py:80} INFO - Job 41: Subtask accurate
[2022-07-27 15:02:22,599] {task_command.py:371} INFO - Running <TaskInstance: dag.accurate manual__2022-07-27T14:58:04.465454+00:00 [running]> on host c054a673cb8f
[2022-07-27 15:02:22,768] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-07-27T14:58:04.465454+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-27T14:58:04.465454+00:00
[2022-07-27 15:02:22,771] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-27 15:02:22,772] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-07-27 15:02:22,799] {subprocess.py:85} INFO - Output:
[2022-07-27 15:02:22,803] {subprocess.py:92} INFO - accurate
[2022-07-27 15:02:22,804] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-27 15:02:22,966] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dag, task_id=accurate, execution_date=20220727T145804, start_date=20220727T150222, end_date=20220727T150222
[2022-07-27 15:02:23,045] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-27 15:02:23,129] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
